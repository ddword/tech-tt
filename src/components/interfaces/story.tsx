export interface IStory {
    by?: string,
    descendants?: number,
    id: number,
    kids: number[],
    score: number,
    text: string,
    title: string,
    time?: number,
    type: string,
    url?: string
}