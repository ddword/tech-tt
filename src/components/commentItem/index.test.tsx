import React from "react";
import { shallow, mount } from "enzyme";
import {IComment} from '../interfaces/comment';
import CommentItem from './';

/*type Props = {
    comment: Partial<IComment>
}*/
const mockComment: Partial<IComment> = {
    id: 18701,
    parent: 101,
    text : "Aw shucks, guys ... you make me blush with your compliments.<p>Tell you what, Ill make a deal: I'll keep writing if you keep reading. K?",
    time : 1314211127,
    type: 'comment',
}

test('renders test react import of CommentItem', () => {
    const wrapper = shallow(<CommentItem comment={mockComment}/>);
    expect(wrapper).toMatchSnapshot();
    //expect(link).toBeInTheDocument();
});