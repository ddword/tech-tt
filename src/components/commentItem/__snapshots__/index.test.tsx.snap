// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders test react import of CommentItem 1`] = `ShallowWrapper {}`;
